README.md

```diff
@@ -574,3 +587,137 @@ WARNING: null
 12:00:02 Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
 12:01:06 Retrying upload after: java.net.SocketException: Broken pipe (Write failed)
 ```
+
+# Member must have length greater than or equal to 20
+
+If you use an invalid IAM Role ID on your credentilas, you would see the following error, 
+you have to set a proper ARN specifying the IAM role to assume. The format should be something like: 
+"arn:aws:iam::123456789012:role/MyIAMRoleName".
```

Comment:
I added screen captures of the errors when you make a configuration validation.

Reply:
I am asking whether documentation like this could be deleted and replaced by code in the validation that displays the real problem and suggested resolution in the UI. Not requesting that this be done _now_—just asking whether it is possible/planned.

Reply:
yep, the validator catch all the exceptions so you can process the exceptions to be more user friendly